applications/vanilla/controllers/class.moderationcontroller.php

```diff
@@ -448,26 +446,9 @@ public function confirmDiscussionMoves($DiscussionID = null) {
 
             // Update recent posts and counts on all affected categories.
             foreach ($AffectedCategories as $categoryID => $counts) {
-                $CategoryModel->refreshAggregateRecentPost($categoryID, true);
```

Comment:
Don't we still need to update the `LastDiscussionID` fields for the categories in the path we're moving to, as well as the ones we're moving away from?

Reply:
From what I see in old code - that was not there.

Reply:
Good point. I need to recheck this stuff.